pkg/virt-config/virt-config.go

```diff
@@ -85,7 +85,8 @@ const (
 	DefaultVirtWebhookClientQPS           = 200
 	DefaultVirtWebhookClientBurst         = 400
 
-	DefaultMaxHotplugRatio = 4
+	DefaultMaxHotplugRatio             = 4
+	DefaultVmRolloutStrategyLiveUpdate = false
```

Comment:
That commit is incredible, thank you so much for it!!
I went back-and-forth on the virt-config bits, what would you suggest, knowing that we'd prefer to keep structs instead of a string?
`DefaultVmRolloutStrategy = v1.RolloutStrategyLiveUpdate{}` doesn't compile...